test/vecelement.jl

```diff
@@ -120,3 +120,9 @@ for T in (Float64, Float32, Int64, Int32)
         @test b == result
     end
 end
+@testset "vecelement overalignment" begin
```

Comment:
I do think those kinds of test are better for llvmpasses, 
since it is easy to match the wrong align here.

Reply:
Any suggestions?